app/views/reductions.html

```diff
@@ -0,0 +1,105 @@
+{% extends "includes/layout.html" %} 
+
+{% block content %}
+<main id="content" role="main">
+    {% include "includes/breadcrumbs.html" %}
+    
+    {% include "includes/validation-error-messages.html" %} 
+    
+    {% include "includes/title-subtitle.html"%}
+    
+    {% include "includes/sub-nav.html" %} 
+    
+    {% if successText %}
+    <div class="message message--success">{{ successText }}</div>
+    {% endif %}
+
+    <a href="/offender-manager/{{ linkId }}/add-reduction" class="button">Add a reduction</a>
+    
+    <h2 class="heading-large">Active<span class="visuallyhidden"> reductions</span></h2>
+
+    <table class="sln-table-reductions">
+        <thead>
+            <tr>
+                <th id="active_type">Type</th>
+                <th id="active_hours">Hours</th>
+                <th id="active_start_date">Start date</th>
+                <th id="active_end_date">End date</th>
+                <th></th>
+            </tr>
+        </thead>
+
+        <tbody class="">
+            {% if newReduction %}
+            <tr>
+                <td headers="active_type">{{ newReduction.reason }}</td>
+                <td headers="active_hours">{{ newReduction.hours }}</td>
+                <td headers="active_start_date">{{ newReduction.startDate }}</td>
+                <td headers="active_end_date">{{ newReduction.endDate }}</td>
+                <td><a href="/om/{{omID}}/reductions/234">View</a></td>
```

Comment:
234?

Reply:
ignore this, it will be addressed for the upcoming view story.

Reply:
Can we update it to link back to the view reductions page or something then? Rather than having a link that won't work in `master`

Reply:
?

Reply:
Going to leave this as is now. Doesn't appear on the frontend and will be fixed in the next story